source/Issue.ts

```diff
@@ -40,4 +45,156 @@ export class IssueModel extends Stream<Issue, IssueFilter>(ListModel) {
         }
         this.totalCount = count;
     }
+
+    /**
+     * Create or update an issue, with support for Copilot assignee
+     *
+     * @see {@link https://docs.github.com/en/rest/issues/issues#create-an-issue}
+     * @see {@link https://docs.github.com/en/rest/issues/issues#update-an-issue}
+     */
+    async updateOne(data: Partial<NewData<Issue>>, id?: number) {
+        const { assignees, ...rest } = data;
+        const assigneeList = assignees as string[] | undefined;
+        const humanAssignees = assigneeList?.filter(login => login !== 'copilot');
+        const hasCopilotAssignee =
+            assigneeList && assigneeList.length !== (humanAssignees?.length || 0);
+
+        const issueData: Partial<NewData<Issue>> = {
+            ...rest,
+            ...(humanAssignees && { assignees: humanAssignees as any })
+        };
+
+        const issue = id ? await super.updateOne(issueData, id) : await super.updateOne(issueData);
+
+        if (hasCopilotAssignee && issue) await this.assignIssueToCopilot(issue);
```

Comment:
```suggestion
        const issue = await super.updateOne(issueData, id);

        if (hasCopilotAssignee) await this.assignIssueToCopilot(issue);
```
创建成功一定会返回非空数据，否则就会抛错。